scripts/lib/mkcloud-dirmaint.sh

```diff
@@ -3,6 +3,8 @@
 # For more information,
 # see http://www.vm.ibm.com/related/dirmaint/overview.html
 
+. ~/.lxccfg
```

Comment:
there needs to be a sanity check for this file existing (e.g. in sanity_checks hooks) with a useful error message..